proxy/src/main/resources/com/velocitypowered/proxy/l10n/messages_hu_HU.properties

```diff
@@ -0,0 +1,62 @@
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+
+velocity.error.already-connected=Már kapcsolódva vagy ehhez a szerverhez!
+velocity.error.already-connected-proxy=Már kapcsolódva vagy ehhez a proxyhoz!
+velocity.error.already-connecting=Már megpróbálsz kapcsolódni egy szerverhez!
+velocity.error.cant-connect=Nem lehet csatlakozni a(z) {0} szerverhez: {1}
+velocity.error.connecting-server-error=Nem tudunk csatlakoztatni a(z) {0} szerverhez. Kérlek próbáld újra később.
+velocity.error.connected-server-error=A kapcsolatod a(z) {0} szerverhez hibába ütközött.
+velocity.error.internal-server-connection-error=Egy belső szerver hiba történt.
+velocity.error.logging-in-too-fast=Túl gyorsan próbálsz csatlakozni, kérlek próbáld újra később.
+velocity.error.online-mode-only=Nem vagy belépve a  Minecraft profilodba. Ha mégis be vagy lépve, kérlek próbáld újraindítani a Minecraft kliensedet.
+velocity.error.player-connection-error=Egy belső hiba keletkezett a kapcsolatodban.
+velocity.error.modern-forwarding-needs-new-client=Ez a szerver csak a Minecraft 1.13 és afölötti verziókkal kompatibilis.
+velocity.error.modern-forwarding-failed=A szerver ahonnan kapcsolódsz nem küldött modern átirányítási kérelmet a proxy felé. Bizonyosodj meg róla, hogy a szerver be van állítva a modern Velocity átirányítás használatára.
+velocity.error.moved-to-new-server=Ki lettél rúgva a(z) {0} szerverről: {1}.
+velocity.error.no-available-servers=Jelenleg nincs elérhető szerver ahová csatlakoztatni tudnánk. Kérlek próbáld újra később, vagy lépj kapcsolatba egy adminisztrátorral.
+
+# Commands
+velocity.command.generic-error=Egy hiba lépett fel a parancs futtatása során.
+velocity.command.command-does-not-exist=Ez a parancs nem létezik.
+
+velocity.command.players-only=Csak játékosok futtathatják le ezt a parancsot.
+velocity.command.server-does-not-exist=A megadott szerver, ( {0} ) nem létezik.
+
+velocity.command.server-current-server=Jelenleg a(z) {0} szerverhez vagy kapcsolódva.
+velocity.command.server-too-many=Túl sok szerver van beállítva. Használd a tab parancs befejező funkciót, hogy megnézd az összes elérhető szervert.
+velocity.command.server-available=Elérhető szerverek:
+velocity.command.server-tooltip-player-online={0} játékos online
+velocity.command.server-tooltip-players-online={0} játékos online
+velocity.command.server-tooltip-current-server=Jelenleg ehhez a szerverhez vagy kapcsolódva
```

Comment:
Esetleg "Jelenleg ezen a szerveren tartózkodsz"?

Reply:
Részletkérdés, szerintem nem annyira lényeges, a többiek mit gondolnak?

Reply:
Feltételezem, hogy ez a file amint meg lesz az új verzió és aki magyarként használja majd magának beletud javítani ha valami éppenséggel nem tetszik benne. Így tehát ez már szőrszálhasogatás kb.

Reply:
Itt nem is arról van szó hogy tudja magának editelni, hanem arról, hogy az alap fordítás mennyire magyaros, illetve pontos.